lib/mountain_view/example.yml

```diff
@@ -0,0 +1,10 @@
+
+- :meta: 'information about the component' #optional
+  :stubs: #as many components as you need
+    -
+      :id: 1
+      :title: "iyevuycveuyv"
+      :subtitle: "ebcecbeuc"
+    -
+      :id: 2
+      :title: "Yejvceuuvcue"
```

Comment:
I mage an array to make compatible with previous code, and not to break all the previous components
```ruby
def stubs?
  styleguide_stubs.is_a?(Array) && styleguide_stubs.any?
rescue Errno::ENOENT
  false
end
````
But definitely making it a hash will remove all that `.first` calls. I don't mind making a 'Hash', what do yoy think @kitop